### PR TITLE
Update to apiVersion, simplified fn calls, removed obsolete script

### PR DESCRIPTION
**Pull request by bmoore-msft**
Updated apiVersion
Simplified some fn calls
provide default params
added outputs
remove old PS script

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@bmoore-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: 101-azure-dns-new-zone/metadata.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   "itemDisplayName": "Azure DNS domain hosting example",
   "description": "This template shows how to create a DNS zone within Azure DNS and how to add some record sets to it.",
   "summary": "This template creates a DNS zone within the Azure DNS resource provider.  It then creates a RecordSet containing two DNS A records within that zone.",
-  "githubUsername": "GarethBradshawMSFT",
-  "dateUpdated": "2016-03-21"
+  "githubUsername": "bmoore-msft",
```
Comment (singhkay) [inline 101-azure-dns-new-zone/metadata.json]: @bmoore-msft Did you mean to change this? Are you going to own this going forward?

Comment (bmoore-msft) [inline 101-azure-dns-new-zone/metadata.json]: Probably, not - put it back...

**Comment by azurecla** [comment]
Hi __@bmoore-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
merged

**Pull request merged**